Author: Daniel Baumann <mail@daniel-baumann.ch>
Description:
 Don't try to execute directories as templates should the user have
 such a situation and specify it (wrongly) (Closes: #701689).

diff -Naurp lxc.orig/src/lxc/lxc-create.in lxc/src/lxc/lxc-create.in
--- lxc.orig/src/lxc/lxc-create.in
+++ lxc/src/lxc/lxc-create.in
@@ -284,7 +284,7 @@ fi
 
 if [ ! -z "$lxc_template" ]; then
     # Allow for a path to be provided as the template name
-    if [ -x "$lxc_template" ]; then
+    if [ -x "$lxc_template" ] && [ ! -d "$lxc_template" ]; then
         template_path=$lxc_template
     else
         template_path=${templatedir}/lxc-$lxc_template
